tests/robustness/failpoints.go

```diff
@@ -528,3 +529,35 @@ func (f delayPeerNetworkFailpoint) Name() string {
 func (f delayPeerNetworkFailpoint) Available(config e2e.EtcdProcessClusterConfig, clus e2e.EtcdProcess) bool {
 	return config.ClusterSize > 1 && clus.PeerProxy() != nil
 }
+
+type noSpaceFailpoint struct {
+}
+
+func (f noSpaceFailpoint) Inject(ctx context.Context, t *testing.T, lg *zap.Logger, clus *e2e.EtcdProcessCluster) error {
+	member := clus.Procs[rand.Int()%len(clus.Procs)]
+	for member.IsRunning() {
+		lg.Info("Setting up gofailpoint", zap.String("failpoint", f.Name()))
+		err := member.Failpoints().Setup(ctx, f.Name(), "return")
```

Comment:
over time, this might render the cluster entirely unusable - what's the best way to turn this off for a given member? After x-minutes? Only if there's still quorum?

Reply:
They are a thing, but we run 100x 5s iteration